docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -87,82 +87,93 @@ more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking
+:underline:`Description of intervention`
 
-.. todo::
+The intervention is the supplementation of children from aged **6 months to 23 months** with **LNS + complementary feeding** (intervention) compared with no intervention (control). The setting of the intervention is the community. 
+
+.. note::
 
-  Fill out the following table with a list of known outcomes affected by the intervention, regardless of if they will be included in the simulation model or not, as it is important to recognize potential unmodeled effects of the intervention and note them as limitations as applicable.
+  - Most studies included children aged six months to 18 months (Adu- Afarwuah 2007; Adu-Afarwuah 2016; Ashorn 2015; Bisimwa 2012; Christian 2015; Hess 2015; Iannotti 2014; Kumwenda 2014; Mangani 2015; Matias 2017; Phuka 2008; Siega-Riz 2014). 
+  - Four studies included children aged six to 24 months (Dewey 2017; Olney 2018; Luby   2018; Null 2018) and 
+  - one study included children aged six to 36 months (Huybregts 2012). 
+  - Four included studies enrolled pregnant women and provided lipid-based nutrient supplements (LNS) plus complementary feeding during pregnancy and post-partum, followed by infant supplementation at six months of age (Adu-Afarwuah 2016; Ashorn 2015; Dewey 2017; Olney 2018). However, Dewey 2017 had an intervention arm in which only children were supplemented, hence we only used the data from that arm in the analysis in this review. The other studies provided LNS plus complementary feeding to children after six months of age. * The review conducted a sensitivity analysis by removing trials that also supplemented pregnant women with LNS in addition to children (Adu-Afarwuah 2016; Ashorn 2015), and found no significant difference in the outcome (Analysis 1.3; Analysis 1.4 in Das DAS_Cochrane_Review_2019).
 
-  The table below provides example entries for large scale food fortification with iron.
 
-.. list-table:: Affected Outcomes
+Please see this memo for a summary of the studies and the effect sizes :download:`SQ-LNS interventions memo<sqlns_memo_das2019.docx>`
+  
+.. list-table:: Affected Outcomes by intervention
   :widths: 15 15 15 15
   :header-rows: 1
 
   * - Outcome
     - Effect
     - Modeled?
     - Note (ex: is this relationship direct or mediated?)
-  * - Hemoglobin concentration
-    - Increases population mean
+  * - Moderate wasting
+    - Reduces prevalence
     - Yes
-    - 
-  * - Malaria
-    - Increases incidence rate
+    - Direct
+  * - Moderate stunting
+    - Reduces prevalence
+    - Yes
+    - Direct
+  * - Severe stunting
+    - Reduces prevalence
+    - Yes
+    - Direct
+  * - Moderate underweight
+    - Reduces prevalence
     - No
-    - 
+    - Direct
+  * - Anaemia
+    - Reduces prevalence
+    - No
+    - Direct
+
 
 Baseline Coverage Data
 ++++++++++++++++++++++++
 
-.. todo::
-
-  Document known baseline coverage data, using the table below if appropriate
+No baseline coverage of SQ-LNS
```

Comment:
Does this mean "no data" or in the baseline/reference scenario, coverage is 0.0%?

Reply:
The baseline coverage in 0.0%.